static/webtorrent.html

```diff
@@ -14,9 +15,11 @@
         height: 140px;
       }
     </style>
+    <script>
+      require('../build/renderer/webtorrent.js')
```

Comment:
Doing that breaks `require()`. If we want `<script ...>` to work, we either have to:
* Write ugly things like `require('../build/renderer/<etc>')` in the actual source files
* Move `main.html` to be next to `main.js`

Doing it this way lets us separate `src/` for JS sources only and `static/` for HTML, CSS, etc